### PR TITLE
[master<860] More info about Docker

### LINKED ISSUE #860: Give an explanation how Memgraph architecture works within Docker
image
something like this

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

I've added some info about how the Memgraph environment looks like in Docker image

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [ ] Documentation fix
- [ ] New documentation page 
- [ ] Base PR for the release
- [x] Documentation improvements
- [ ] Other (please describe):

### Related PRs and issues

Closes #860

### Checklist:

- [ ] I checked all content with Grammarly
- [ ] I performed a self-review of my code
- [ ] I made corresponding changes to the rest of the documentation
- [ ] The build passes locally
- [ ] My changes generate no new warnings or errors

### CODE REVIEW

**Review by antejavor — CHANGES_REQUESTED** [review]
Except for the comments, take a look at the paths, we are mixing the Windows and Linux paths, lines 120 - 140.

File: docs/how-to-guides/work-with-docker.md [hunk 1]
```diff
@@ -105,24 +97,25 @@ image and you want to connect to both instances using the Memgraph Lab
...
+docker run -it -p 7444:7444 -p 3001:3000 memgraph/memgraph-platform
 ```
 
 To connect to the first instance, you should open Memgraph Lab in your browser
-at `localhost:3000`, but the second instance is at `localhost:3001`.
+at `localhost:3000`, but the second instance is reachable at `localhost:3001`.
 
 #### Specify volumes
 
 Specifying a volume creates a copy of a directory inside the Docker container as
```
Comment (vpavicic) [inline docs/how-to-guides/work-with-docker.md:115]:
```suggestion
Specifying a volume syncs the specified directory inside the Docker container as
```

File: docs/how-to-guides/work-with-docker.md [hunk 2]
```diff
@@ -146,6 +139,25 @@ Logs can usually be found in
...
+with the following `-v` flag.
+
+```
+docker run -it -p 7687:7687 -p 7444:7444 -p 3000:3000  -v "C:/data":/usr/lib/memgraph/data memgraph/memgraph-platform
+```
+
+You can use bind mounts to transfer durability files such as snapshot or wal
+files inside the container to restore data, or CSV files you will use to import
+data with `CSV LOAD` clause. 
+
```
Comment (vpavicic) [inline docs/how-to-guides/work-with-docker.md:160]:
```suggestion

Bind mounts do not handle data permissions which could cause issues with permissions. 
```

**Comment by vpavicic** [comment]
> Except for the comments, take a look at the paths, we are mixing the Windows and Linux paths, lines 120 - 140.

I will open a new issue for this because i need to fix it at many places and need to think about it a bit, how to handle it :)

**Pull request merged**